oak_functions/examples/fuzzable/module/src/tests.rs

```diff
@@ -16,18 +16,20 @@
 use crate::proto::{
     instruction::InstructionVariant, Instruction, Instructions, Panic, WriteResponse,
 };
-use maplit::hashmap;
+use maplit::btreemap;
```

Comment:
Is this related to making the crate no-std? What is the performance implication? Or does it not matter since we expect these maps to be small?

Reply:
Yes, there is no hashmap in the alloc crate as the default implementation relies on a std source of randomness. We could switch to a different impelementation (such as hashbrown), but these would only contain the various metrics buckets so I don't expect it to contain more than 5 or 10 items. I think a btreemap is probably more efficient for that number of entries.